include/volume/cooling_balls.h

```diff
@@ -65,9 +67,11 @@ NT vol_cooling_balls(Polytope &P, UParameters &var, AParameters &var_ban, std::p
     // Save the radius of the Chebychev ball
     var.che_rad = radius;
     // Move the chebychev center to the origin and apply the same shifting to the polytope
-    VT c_e = Eigen::Map<VT>(&c.get_coeffs()[0], c.dimension());
+
+    VT c_e = c.getCoefficients();
     P.shift(c_e);
 
+
```

Comment:
needed?

Reply:
I can avoid copying the vector by passing c.getCoefficients() in P.shift(). Is that what you mean?

Reply:
sure, if you can avoid that copy it would be nice